examples/redis-todo/db/models/userModel.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model } = require('mongoose')
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcryptjs')
+
+const userSchema = new Schema({
+  name: { type: String, required: true },
+  username: { type: String, unique: false, required: true }, // TODO: change this to true
+  email: { type: String, unique: false, required: true },
+  password: { type: String, required: true }
+}, { timestamps: true, versionKey: false })
+
+userSchema.methods.toJSON = function () {
+  const user = this.toObject() // this = user
+  delete user.password
```

Comment:
Generally speaking, I'd prefer if `password` were stored outside the user schema. Here's some more thoughts: https://thecodebarbarian.com/thoughts-on-user-passwords-in-rest-apis

Reply:
I had separated password from user schema to new password schema